packages/storybook/stories/CallComposite/CallCompositeDocs.tsx

```diff
@@ -10,6 +10,10 @@ const containerText = require('!!raw-loader!./snippets/Container.snippet.tsx').d
 const customDataModelExampleContainerText =
   require('!!raw-loader!./snippets/CustomDataModelExampleContainer.snippet.tsx').default;
 
+const mobileViewSnippet = `
+<CallComposite options={{ mobileView: true }} />
```

Comment:
Does automation compile-check this?

Reply:
It doesn't but I wanted to keep the snippet ultra small, happy to make this longer if we want the compile check but will require defining an adapter and having imports etc.